db/migrate/20170319180638_remove_mentor_fav_from_applications.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveMentorFavFromApplications < ActiveRecord::Migration[5.0]
+  def up
+    remove_column :applications, :mentor_fav
```

Comment:
I'm not super familiar with the data in the database. just as an idea.
should we can keep this for a while? maybe it's helpful to partially backfill the data?
if yes, I suggest renaming it to do `deprecated_mentor_fav` or something similar.